AmplifyTools/AmplifyXcode/Sources/AmplifyXcodeCore/Commands/CommandImportConfig.swift

```diff
@@ -17,7 +17,7 @@ enum ImportConfigTasks {
 
         return .failure(AmplifyCommandError(
                             .folderNotFound,
-                            error: nil,
+                            errorDescription: "We couldn't find an Amplify project at \(environment.basePath).",
                             recoverySuggestion: "Please run `amplify init` to initialize an Amplify project."))
```

Comment:
Avoid "we" in our error messages, prefer more terse constructions. Similarly, we avoid "please" in messages & constructions, and prefer being direct and prescriptive.
```suggestion
                            errorDescription: "Amplify project not found at \(environment.basePath).",
                            recoverySuggestion: "Run `amplify init` to initialize an Amplify project."))
```

Reply:
Will do, thanks